db/migrate/20201014060724_add_column_to_worksheet_reviews.rb

```diff
@@ -0,0 +1,5 @@
+class AddColumnToWorksheetReviews < ActiveRecord::Migration[6.0]
```

Comment:
best practice when you have several migrations that haven't been merged is to remove the extra files and merge the content into a single migration
it's more efficient in heroku to run a single migration than to run 4 different ones